test/cmdlineTests/exp_base_literal/output

```diff
@@ -208,78 +208,157 @@ object "C_81" {
             }
 
             function fun_f_80(var_a_4, var_b_6, var_c_8, var_d_10) -> var__13, var__15, var__17, var__19 {
+                /// @origin exp_base_literal/input.sol:160,164
```

Comment:
Also, was thinking about adding some kind of notation to the beginning of a file to state what is the origin (in this case it is solidity source)